web/gds-user-ui/src/modules/dashboard/member/Components/MemberNetworkSelect.tsx

```diff
@@ -0,0 +1,34 @@
+import { FormControl, FormLabel, Select } from "@chakra-ui/react";
+import { Trans } from "@lingui/macro";
+import { useState } from "react";
+import { DirectoryType, DirectoryTypeEnum } from "../memberType";
+import { getMembersService } from "../service";
+
+const MemberSelectNetwork = () => {
+  const [selectNetwork, setSelectNetwork] = useState("");
+
+  const handleSelectNetwork = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    e.preventDefault();
+    return setSelectNetwork(e.target.value);
+  };
+
+  getMembersService(selectNetwork as DirectoryType);
```

Comment:
Adding the network value to `getMembersService` here meant that I don't need to store the value to local storage. If handling this a different way is preferred, let me know.